app/javascript/components/Places/Form/validationSchema.js

```diff
@@ -0,0 +1,12 @@
+import * as Yup from 'yup'
+
+const schema = Yup.object().shape({
+  name: Yup.string().required('This field is required'),
```

Comment:
I would suggest to use `I18n` here to show correct translations.
Like:
```
import { I18n } from 'components/TranslationsProvider'
...
name: Yup.string().required(() => I18n.t('activerecord.errors.places...')
```
Can't pull translation key straight out of my head, but I'm sure it is easy to find